schematics/ng-add/index.ts

```diff
@@ -140,6 +146,8 @@ function updateProjectNewAngular(context: SchematicContext, clientProject: Works
   buildTarget.options.main = join(clientProject.root, 'src/main.single-spa.ts');
   (buildTarget.options as CustomWebpackBuilderOptions).customWebpackConfig = {
     path: join(clientProject.root, 'extra-webpack.config.js'),
+    libraryName: projectName,
+    libraryTarget: 'umd',
```

Comment:
By changing this line, it is possible to change the default libraryTarget (perhaps to system), without it being a breaking change, as previously created apps would still use fallback default in webpack/index.ts.